Source/OxyPlot/Rendering/RenderContext/RenderingExtensions.cs

```diff
@@ -93,6 +93,25 @@ M3    M3
                 return;
             }
 
+            if (rc.SetClip(clippingRectangle))
+            {
+                rc.DrawLine(points, stroke, strokeThickness, dashArray, lineJoin, aliased);
+                rc.ResetClip();
```

Comment:
After small research I think you are right. It's better to call SetClip in, for example, LineSeries render. It seems clipped lines coords mostly used in annotation classes for label rendering. I think this PR can broke them. I'll try search for DrawClippedLine calls where last 2 arguments (output buffer and callback) not used and surround them with SetClip/ResetClip. I try to open new PR in a couple of days.